virTemplate.md submission for final project

jheeswyk opened the pull request.

jheeswyk committed:
virTemplate and images submission

jheeswyk committed:
images

jheeswyk committed:
update

ababaian commented on template/img/alignment.PNG:
please place all images in a specific folder for your Obelisk and not in the `template` folder but in the `virus` folder. See other submissions from the last year for examples of how filepaths should look like. Amend the paths in the Markdown submission as well so it works.

ababaian commented on template/img/alignment.PNG:Zone.Identifier:
remove empty file.

ababaian commented on template/virTemplate.md:
Please don't modify the `virTemplate.md` file directly in your PR. Instead copy the file and re-name it to match your submission. See how the files are arranged in other submission.s

ababaian commented on template/virTemplate.md:
```diff
@@ -1,40 +1,140 @@
 
-{Q2 answer ...}
 
 ### Virus Genome {Q3}
```
Amend out the "Default Titles" from the document. Your descriptive titles are much better throughout.

ababaian commented on template/virTemplate.md:
```diff
@@ -1,40 +1,140 @@
 ### Virus Genome {Q3}
 
-{Insert your Virus Genome Map Figure and figure legend here}
+![Fig. 1A: ORFs of Clostridiae-associata centralis as determined by ORFfinder, with nucleotide scale bar. Created with R package gggenes based on GitHub PuravG/Computational-Virus-Discovery](img/genome.png)
```
if the "Figure Legend" is in the title, it won't be displayed, The square-bracket text here is the "title" shown if the image does not load. Instead put the text in italics underneath the figure.

ababaian reviewed (changes requested):
Submission looks really cool :) I'm looking forward to diving into it. Please amend the PR to fix the formatting requirements.

jheeswyk committed:
fixed formatting

jheeswyk committed:
Delete template/.Rhistory

jheeswyk committed:
Delete template/img directory

jheeswyk commented:
Hi Artem,
I edited my submission file and renamed it to Clostridiae-associata_centralis.md, changed the filepaths for it and my images as requested, and reverted virTemplate.md to the unfilled version. Please let me know if everything looks good!

ababaian commented on template/.Rhistory:
Please remove/delete dotfiles from the repo.
